gui/qt/main_window.py

```diff
@@ -1484,7 +1483,7 @@ def fee_cb(dyn, pos, fee_rate):
         grid.addWidget(self.token_type_combo, 7, 1)
 
 
-        if self.config.get('enable_slp') == False:
+        if self.config.get('enable_slp', True) == False:
```

Comment:
OK, here is me being pedantic: what you did works and is fine.. But you should really get in the habit of avoiding the `==True` or `==False` thing because it's a bit harder to read and not as 'standard'.

`if not self.config.get('enable_slp', True)` rather than `if self.config.get('enable_slp', True) == False` is much easer to read.

Aside from readability it's a good habit not to compare to exactly False/True in case you want to work with more types  -- where you may get an int, (0 evaluates to False in a bool context, nonzero true) or a list or a dict etc etc..